eq-author/src/App/section/Design/index.js

```diff
@@ -98,6 +100,19 @@ export const UnwrappedSectionRoute = (props) => {
     onDeleteSection(sectionId);
   };
 
+  const redirectPage = () => {
+    if (section === null && questionnaire !== null) {
+      return (
+        <RedirectRoute
+          from={"/q/:questionnaireId/section/:sectionId/design"}
+          to={`/q/:questionnaireId/section/${questionnaire.sections[0].id}/design`}
+        />
+      );
+    }
+  };
+
+  const { questionnaire } = useQuestionnaire();
```

Comment:
```suggestion
  const redirectPage = () => {
    if (section === null) {
      return (
        <RedirectRoute
          from={"/q/:questionnaireId/section/:sectionId/design"}
          to={`/q/:questionnaireId/section/${questionnaire?.sections[0]?.id}/design`}
        />
      );
    }
  };

  const { questionnaire } = useQuestionnaire();
```

Opening a deleted section in a separate tab causes a crash without ?s